pkg/catalog/diagram.go

```diff
@@ -228,14 +265,12 @@ func (p *Generator) CreateTypeDiagram(app *sysl.Application, typeName string, t
 func CreateFileName(dir string, elems ...string) (string, string) {
 	absolutefileName := path.Join(append([]string{dir}, Map(elems, SanitiseOutputName)...)...)
 	relativefileName := strings.Replace(absolutefileName, dir+"/", "", 1)
-	absolutefileName = strings.ToLower(absolutefileName)
-	relativefileName = strings.ToLower(relativefileName)
```

Comment:
strings.ToLower is needed so that there aren't filename clashes

Reply:
👍 Dropped that commit